src/app/test/service/assert.ts

```diff
@@ -18,10 +18,8 @@ export function assert<T>(actual: T, expected: T, message?: string, sanitize: bo
 /**
  * Removes port from IP address. Sometimes IP address is returned with port, which is not needed.
  * */
-function removePortFromIp(ip: string) {
```

Comment:
what is the reason behind this change?

Reply:
Got it. The output of this function when an IPv6 with port is used contains `[]` characters. For instance:
```
let ip = '[1050:0000:0000:0000:0005:0600:300c:326b]:3000'
console.log(removePortFromIp(ip)) // prints [1050:0000:0000:0000:0005:0600:300c:326b]
```
Is this expected or a mistake?

Reply:
This is expected, as a result all IPv6 and IPv4 tests passes without the problem of brackets.